src/components/Auth/PwdFindForm.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import * as S from 'components/Auth/styles';
+import Button from 'components/Shared/Button';
+import AuthNumber from 'components/Shared/AuthNumber';
+
+const PwdFindForm = () => {
+  return (
+    <div>
+      <S.Title>비밀번호</S.Title>
+      <S.StyledInput placeholder="아이디 입력"></S.StyledInput>
+      <S.StyledInput placeholder="이메일 입력"></S.StyledInput>
```

Comment:
React JSX 는 안에 content 가 없는 태그를 자체적으로 닫을 수 있습니다.
`self-closing tag` 라는 용어로 불립니다

```suggestion
      <S.StyledInput placeholder="아이디 입력" />
      <S.StyledInput placeholder="이메일 입력" />
```

Reply:
다른것들도 안되있는게 있어서 더 찾아서 수정했습니다!

Reply:
a3044e2